server/database/data/build.sql

```diff
@@ -0,0 +1,32 @@
+BEGIN;
+
+DROP TABLE IF EXISTS USERS CASCADE;
+DROP TABLE IF EXISTS PRODUCTS CASCADE;
+DROP TABLE IF EXISTS CART CASCADE;
+
+CREATE TABLE IF NOT EXISTS USERS(
+    USER_ID SERIAL PRIMARY KEY,
```

Comment:
attributes should be in lowercase

Reply:
I have resolved this issue. Thanks Elham.